docs/azmcp-commands.md

```diff
@@ -48,6 +48,21 @@ azmcp cosmos database container item query --subscription <subscription> \
                        [--query "SELECT * FROM c"]
 ```
 
+### Sql Operations
+```bash
+# List SQL Servers in a subscription
+azmcp sql server list --subscription <subscription>
+
+# List SQL Databases in a SQL Server
+azmcp sql server database list --subscription <subscription> --server-name <server-name>
```

Comment:
why not just use azmcp sql database list which matches cli?

Reply:
I followed the CosmosDB pattern for azcmp, which puts container under database (the equivalent for Sql is database under server).  It's the same in the Az CLI for CosmosDB, its:

az cosmosdb sql database
az cosmosdb sql container

But their azcmp command  is

azcmp cosmodb database container

The equivalent for SQL in the az cli is (a db is contained within a server)

az sql server
az sql db

So to be the same as azcmp cosmosdb, for sql we end up with

azcmp sql server database

(I expanded out the word "database" from "db" to again be the same as CosmosDB in azcmp)

Is it correct for sql to follow the CosmosDB pattern in azcmp, or to follow the pattern in Az CLI (which cosmosdb doesn't follow)

Reply:
It seems to me that CosmosDB follows the same pattern as what's used in the CLI,  https://learn.microsoft.com/en-us/cli/azure/cosmosdb?view=azure-cli-latest

whereas SQL uses a different pattern in your CLI command. :)

Reply:
Can you give an example.  I have given you an example of how SQL follows the same pattern as Cosmos, again:

Cosmos:

azmcp cosmos database container item query

Sql:

azmcp sql server database query

Reply:
This is the container command in the Cosmos CLI:
![image](https://github.com/user-attachments/assets/b1add394-2621-462a-ac56-95f4d5c51cae)

And here’s your command:
![image](https://github.com/user-attachments/assets/2ac17300-7c86-4327-85ab-af71c7abdeca)

To stay consistent with the existing pattern, I don't think we need "server" in your command.
We could simply use:
"azmcp sql database query".

Are you planning to introduce something like:
azmcp sql xxx database query?

If not, I believe "azmcp sql database query" is sufficient.

Reply:
Thanks for the information.

In CosmosDB, operations can be performed directly on databases:
![image](https://github.com/user-attachments/assets/18da1505-063f-464e-a315-8ee2a2df7ca5)

Do you currently have, or plan to add, operations on servers as well?
(I couldn’t find any server-related operations in the CLI.)